test/unit-tests/isDisabled.test.js

```diff
@@ -13,134 +13,72 @@ let {
 } = require('../../lib/taiko');
 let { createHtml, removeFile, openBrowserArgs, resetConfig } = require('./test-util');
 let test_name = 'tsesr';
```

Comment:
I don't understand this string and it's usage in the next line

Reply:
seems a typo.. will push a fix

Reply:
done 👍